plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/AppEngineDeployPreferencesPanel.java

```diff
@@ -418,11 +408,8 @@ public void widgetSelected(SelectionEvent event) {
         new ProjectSelectorSelectionChangedListener(accountSelector,
                                                     projectRepository,
                                                     projectSelector));
-    filterField.addModifyListener(new ModifyListener() {
-      @Override
-      public void modifyText(ModifyEvent event) {
-        projectSelector.setFilter(filterField.getText());
-      }
+    filterField.addModifyListener(event -> {
```

Comment:
We don't need the braces, right?

Reply:
Since the line is physically long, I am tempted to have the braces to look more like the old-fashioned method.